imageroot/actions/list-certificates/validate-output.json

```
@@ -1,22 +1,72 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "list-certificates output",
-    "$id": "http://schema.nethserver.org/samba/list-certificates-output.json",
+    "$id": "http://schema.nethserver.org/traefik/list-certificates-output.json",
     "description": "Return a list of requested certificates fqdn",
     "examples": [
-	    ["example.com"],
-	    []
+        [
+            "foo.domain.com",
+            "nextcloud.domain.com",
+            "webserver2.domain.com"
+        ],
+        [
+            {
+                "fqdn": "foo.domain.com",
+                "type": "internal",
+                "obtained": true
+            },
+            {
+                "fqdn": "nextcloud.domain.com",
+                "type": "route",
+                "obtained": true
+            },
+            {
+                "fqdn": "webserver2.domain.com",
+                "type": "custom",
+                "obtained": true
+            }
+        ]
     ],
     "type": "array",
-    "items": {
-	    "oneOf":[{
-		    "type": "object",
-		    "title": "A certificate expanded"
-	    },
-	    {
-		    "type": "string",
-		    "format": "hostname",
-		    "title": "A fully qualified domain name"
-	    }]
-    }
-}
+    "anyOf": [
+        {
+            "minItems": 1,
+            "items": {
+                "type": "string",
+                "format": "idn-hostname"
+            }
+        },
+        {
+            "minItems": 1,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "fqdn": {
+                        "type": "string",
+                        "format": "idn-hostname"
+                    },
+                    "type": {
+                        "type": "string",
+                        "enum": [
+                            "internal",
+                            "custom",
+                            "route"
+                        ]
+                    },
+                    "obtained": {
+                        "type": "boolean"
+                    }
+                },
+                "required": [
+                    "fqdn",
+                    "type",
+                    "obtained"
+                ]
+            }
+        },
+        {
+            "minItems": 0,
+            "maxItems": 0
+        }
+    ]
```

There is no need to use `minItems`/maxItems` if we use `anyOf`
```suggestion
    "anyOf": [
        {
            "items": {
                "type": "string",
                "format": "idn-hostname"
            }
        },
        {
            "items": {
                "type": "object",
                "properties": {
                    "fqdn": {
                        "type": "string",
                        "format": "idn-hostname"
                    },
                    "type": {
                        "type": "string",
                        "enum": [
                            "internal",
                            "custom",
                            "route"
                        ]
                    },
                    "obtained": {
                        "type": "boolean"
                    }
                },
                "required": [
                    "fqdn",
                    "type",
                    "obtained"
                ]
            }
        }
    ]
```